### PR TITLE
add RabbitMQ Add-On documentation page

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: pages/03.addons/13.metakube-rabbitmq/default.en.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+| ------ | ----------- | ------------- |
+| CPU Requests | How many milli CPUs should be reserved | 200m |
+| Memory Requests | How much memory should be reserved | 256Mi |
+| CPU Limits | What is the maximum of used milli CPUs | 1000m |
+| Memory Limits | What is the maximum of used memory | 1Gi |
+| Replicas | How many replicas should be running | 2 |
+
+## Getting RabbitMQ Credentials
+
+After the installation of the Add-On you can retrive the RabbitMQ credentials with the following command:
```
Comment (phiphi282) [inline pages/03.addons/13.metakube-rabbitmq/default.en.md]:
```suggestion
After the installation of the Add-On you can retrieve the RabbitMQ credentials with the following command:
```

File: pages/03.addons/13.metakube-rabbitmq/default.en.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+echo "Username: user" && echo "Password: $(kubectl get secret -n syseleven-rabbitmq syseleven-rabbitmq -o jsonpath="{.data.rabbitmq-password}"|base64 --decode)"
+```
+
+## Accessing the RabbitMQ Management UI
+
+You can reach the RabbitMQ Management UI from the cluster detail page. In addition to the standard SysEleven authentication, you have to log into RabbitMQ Management UI itself using your credentials.
+
+## Accessing RabbitMQ from within Kubernetes
+
+When installing RabbitMQ corresponding Kubernetes services are created. You can use Kubernetes mechanisms to reach to Add-On Pods using following services:
```
Comment (phiphi282) [inline pages/03.addons/13.metakube-rabbitmq/default.en.md]:
```suggestion
When installing RabbitMQ corresponding Kubernetes services are created. You can use Kubernetes mechanisms to reach to Add-On Pods using the following services:
```

**Review by phiphi282 — CHANGES_REQUESTED** [review]
Looks pretty good, only found some minor linguistic stuff.

**Comment by furkhat** [comment]
@phiphi282 updated

**Review by phiphi282 — APPROVED** [review]
lgtm

**Comment by eduardostalinho** [comment]
LGTM

**Pull request merged**